src/main/java/com/dev/cinema/service/MovieSessionService.java

```diff
@@ -0,0 +1,11 @@
+package com.dev.cinema.service;
+
+import com.dev.cinema.model.MovieSession;
+import java.time.LocalDate;
+import java.util.List;
+
+public interface MovieSessionService {
+    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
+
+    MovieSession add(MovieSession session);
+}
```

Comment:
Why didn't you add here `getAll() `method and its implementation, if you did it in `MovieSessionDao`?

Reply:
I thought if it is not in the requirements for the `MovieSessionService` so there is not requirement to implement it in the `MovieSessionDao` respectively. I think it is no problem to add of request)